ext/bg/js/backend.js

```diff
@@ -967,6 +944,50 @@ class Backend {
     }
 
     // Utilities
+    async _modifySetting(target) {
+        const optionsContext = target.optionsContext;
+        const options = isObject(optionsContext) ? this.getOptions(optionsContext, true) : this.getFullOptions(true);
```

Comment:
Maybe full options should be explicitly enabled in case the API user forgets to supply context and breaks things

Reply:
Updated. c12887daf3e4b81233a5fdc39af951de53267fe2